### PR TITLE
BrenKt-TRANS-WEB-Domain-configure_MX 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/domains/dns_zone_mx/guide.es-es.md [hunk 1]
```diff
@@ -1,71 +1,86 @@
...
+> - Si el dominio no utiliza los servidores DNS de OVHcloud, deberá editar los registros MX desde el panel que le ofrezca el proveedor que gestione la configuración del dominio.
 >
-> - Si el dominio está registrado en OVHcloud, puede comprobar si utiliza nuestra configuración desde el [área de cliente](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}. Para ello, haga clic en `Dominios`{.action} y seleccione el dominio. A continuación, abra la pestaña `Servidores DNS`{.action}.
+> - Si el dominio está registrado en OVHcloud, compruebe que utiliza nuestra configuración en el [área de cliente](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) de OVHcloud. En la pestaña « `Servidores DNS`{.action} » de la columna izquierda, haga clic en Dominios en la columna izquierda y seleccione el dominio correspondiente, en la pestaña `información general`{.action}. Si en la sección « **Servidores DNS** » aparece el epígrafe `Activo`, usted utiliza los servidores DNS de OVHcloud.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.es-es.md]:
```suggestion
> - Si el dominio está registrado en OVHcloud, compruebe que utiliza nuestra configuración en el [área de cliente](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es) de OVHcloud. En la pestaña « `Servidores DNS`{.action} » de la columna izquierda, haga clic en Dominios en la columna izquierda y seleccione el dominio correspondiente, en la pestaña `Información General`{.action}. Si en la sección "**Servidores DNS**" aparece el epígrafe `Activo`, usted utiliza los servidores DNS de OVHcloud.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.es-es.md [hunk 2]
```diff
@@ -1,71 +1,86 @@
...
+- **(1)** consultar la zona DNS del dominio **mydomain.ovh** y leer los registros **MX**.
+- **(2)** reenviar el mensaje de correo electrónico a la URL del registro **MX** leído.
 
-Dado que es posible configurar varios registros MX para un mismo dominio, es necesario establecer una prioridad para cada uno de ellos. De este modo, los servidores que envían los mensajes de correo electrónico saben a qué servidor deben dirigirlos en primer lugar. Sin embargo, solo es posible añadir registros MX pertenecientes a un mismo proveedor.
+![Correo electrónico](images/email-dns-conf-mx01.png){.thumbnail}
 
-**Editar los registros MX de un dominio es una operación delicada**, ya que una configuración errónea podría deshabilitar la recepción de nuevos mensajes en sus direcciones de correo. Así pues, debe prestar especial atención al modificar los registros MX de un dominio.
+El mensaje se enviará al destino **mx0.mail.ovh.net**, precedido del valor **0**. Este valor se denomina prioridad. El valor más bajo se consulta primero y el más alto, después. Esto significa que la presencia de varios registros permite paliar la falta de respuesta del registro MX de menor prioridad.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.es-es.md]:
```suggestion
El mensaje se enviará al destino **mx0.mail.ovh.net**, precedido del valor **0**. Este valor se denomina prioridad. El valor más bajo se consulta primero y el más alto después. Esto significa que la presencia de varios registros permite paliar la falta de respuesta del registro MX de menor prioridad.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.es-es.md [hunk 3]
```diff
@@ -1,71 +1,86 @@
...
+![Registro MX en la zona DNS](images/mx-records-dns-zone.png){.thumbnail}
 
-Para comprobar más rápidamente si ya existen registros MX, seleccione, utilizando el filtro situado sobre la tabla DNS, el campo de tipo **MX** y acepte para mostrar únicamente los registros DNS MX de su zona DNS.
+- Si los registros MX ya existen y desea modificarlos, haga clic en el botón `...`{.action} a la derecha de cada fila de la tabla correspondiente y seleccione `Modificar el registro`{.action}.
+- Si no hay ningún registro MX, haga clic en el botón `Añadir un registro`{.action} a la derecha de la tabla y seleccione `MX`{.action}. Introduzca la información solicitada en función de la solución de correo elegida:
 
-Para crear un nuevo registro MX, haga clic en el botón `Añadir un registro`{.action} a la derecha de la tabla y seleccione `MX`{.action}. Complete la información solicitada (en función de la solución de correo de que disponga):
+**Si tiene contratada una solución de correo electrónico de OVHcloud**, consulte la información que encontrará en el apartado [Conocer la configuración MX de OVHcloud ](#mxovhcloud).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.es-es.md]:
```suggestion
**Si tiene contratada una solución de correo electrónico de OVHcloud**, consulte la información que encontrará en el apartado [Conocer la configuración MX de OVHcloud](#mxovhcloud).
```

File: pages/web_cloud/domains/dns_zone_mx/guide.it-it.md [hunk 4]
```diff
@@ -1,85 +1,104 @@
...
 
-> [!warning]
+> [!primary]
 >
-> - Se il dominio non utilizza i server DNS OVHcloud, la modifica dei record MX deve essere eseguita dall’interfaccia del provider che ne gestisce la configurazione.
+> - Se il dominio non utilizza i server DNS di OVHcloud, la modifica dei record MX deve essere eseguita dall’interfaccia del provider che gestisce la configurazione del dominio.
 >
-> - Per i domini registrati in OVHcloud è invece possibile verificare la configurazione direttamente dallo [Spazio Cliente](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it){.external}, cliccando sulla scheda `Server DNS`{.action}.
+> - Se il dominio è registrato presso OVHcloud, è possibile verificarne la configurazione attraverso lo [Spazio Cliente](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it), sezione `Server DNS`{.action} > Dominio pertinente > scheda `informazioni generali`{.action}. Se la voce `Attivo` è presente alla voce "**server DNS**", utilizza i server DNS di OVHcloud.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.it-it.md]:
```suggestion
> - Se il dominio è registrato presso OVHcloud, è possibile verificarne la configurazione attraverso lo [Spazio Cliente](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it), sezione `Server DNS`{.action} > Dominio pertinente > scheda `Informazioni Generali`{.action}. Se la voce `Attivo` è presente alla voce "**server DNS**", utilizza i server DNS di OVHcloud.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.it-it.md [hunk 5]
```diff
@@ -1,85 +1,104 @@
...
 
 ## Procedura
 
-### Step 1: conoscere alcune nozioni di base del record MX
+### Informazioni sul ruolo dei record MX 
+
+I record MX (**M**ail **e**Xchange) permettono di collegare il dominio ai server di posta associati al servizio di posta. Ci baseremo su un esempio.
+
+Quando l'indirizzo **sender@otherdomain.ovh** invia un’email a **contact@mydomain.ovh**, il server di invio delle email (**Outgoing mail server**):
+- **(1)** interrogare la zona DNS del dominio **mydomain.ovh** e leggere i record **MX**.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.it-it.md:40]:
```suggestion

- **(1)** interrogare la zona DNS del dominio **mydomain.ovh** e leggere i record **MX**.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.es-es.md [hunk 6]
```diff
@@ -1,71 +1,86 @@
...
 ## Procedimiento
 
-### 1. Los registros MX
+### Comprender la función de los registros MX 
+
+Los registros MX (**M**ail **e**Xchange) permiten asociar su dominio a los servidores de correo de recepción asociados al servicio de correo. Vamos a basarnos en un ejemplo.
 
-Los registros MX permiten asociar un dominio al servidor de un proveedor de correo, como OVHcloud. Cuando una persona envía un mensaje de correo electrónico, el servidor que realiza el envío sabe a qué servidor debe dirigirlo gracias al registro MX.
+Cuando la dirección **sender@otherdomain.ovh** envía un correo electrónico a **contact@mydomain.ovh**, el servidor de envío de correo (**Outgoing mail server**) envía el siguiente mensaje:
+- **(1)** consultar la zona DNS del dominio **mydomain.ovh** y leer los registros **MX**.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.es-es.md:40]:
```suggestion

- **(1)** consultar la zona DNS del dominio **mydomain.ovh** y leer los registros **MX**.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.it-it.md [hunk 7]
```diff
@@ -1,85 +1,104 @@
...
-Per verificare più rapidamente se esistono già record MX, seleziona il record di tipo **MX** con il filtro sopra la tabella DNS e conferma per visualizzare solo i record DNS MX della tua zona DNS.
+![dnsmxrecord](images/mx-records-dns-zone.png){.thumbnail}
 
-Per eseguire questa operazione, clicca su `Aggiungi un record`{.action} a destra della tabella e seleziona `MX`{.action}. In base al servizio email scelto, inserisci le informazioni richieste: 
+- Se esistono record MX e vuoi modificarli, clicca sui tre puntini `...`{.action} a destra di ogni riga della tabella e poi clicca su `Modifica record`{.action}.
+- Se non sono presenti record MX, clicca sul pulsante `Aggiungi un record`{.action} a destra della tabella e seleziona `MX`{.action}. In base alla soluzione email scelta, inserisci le informazioni richieste:
 
-- **se utilizzi una soluzione di posta OVHcloud**, consulta il paragrafo precedente di questa guida: [Step 2: conoscere la configurazione MX di OVHcloud](./#step-2-conoscere-la-configurazione-mx-di-ovh){.external}
+**Se disponi di una soluzione email OVHcloud**, consulta le informazioni fornite nello step "[Conoscere la configurazione MX di OVHcloud ](#mxovhcloud)".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.it-it.md]:
```suggestion
**Se disponi di una soluzione email OVHcloud**, consulta le informazioni fornite nello step "[Conoscere la configurazione MX di OVHcloud](#mxovhcloud)".
```

File: pages/web_cloud/domains/dns_zone_mx/guide.pl-pl.md [hunk 8]
```diff
@@ -1,85 +1,104 @@
 ---
-title: 'Dodanie rekordu MX w konfiguracji domeny'
-excerpt: 'Dowiedz się, jak dodać rekord MX do strefy DNs domeny'
-updated: 2018-05-30
+title: Konfiguracja rekordu MX
+excerpt: Dowiedz się, jak skonfigurować rekord MX dla Twojej domeny w OVHcloud
+Updated: 2023-08-30
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pl-pl.md]:
```suggestion
updated: 2023-08-30
```

File: pages/web_cloud/domains/dns_zone_mx/guide.pl-pl.md [hunk 9]
```diff
@@ -1,85 +1,104 @@
...
 
 ## W praktyce
 
-### Etap 1: podstawowe informacje o rekordzie MX
+### Zrozumienie roli rekordów MX 
+
+Rekordy MX (**M**ail **e**Xchange) umożliwiają powiązanie Twojej domeny z serwerami poczty przychodzącej przypisanymi do Twojej usługi e-mail. Posłużemy się przykładem.
+
+Gdy adres **sender@otherdomain.ovh** wysyła wiadomość na adres **contact@modomain.ovh**, serwer poczty (**Outgoing mail server**) przechodzi:
+- **(1)** odpytywanie strefy DNS domeny **mydomain.ovh** i odczytywanie rekordów **MX**.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pl-pl.md:40]:
```suggestion

- **(1)** odpytywanie strefy DNS domeny **mydomain.ovh** i odczytywanie rekordów **MX**.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.pl-pl.md [hunk 10]
```diff
@@ -1,85 +1,104 @@
...
+![dnsmxrecord](images/mx-records-dns-zone.png){.thumbnail}
 
-Kliknij przycisk `Dodaj rekord`{.action} po prawej stronie tabeli, następnie wybierz `MX`{.action}. Wpisz wymagane informacje w zależności od wybranego rozwiązania poczty elektronicznej:
+- Jeśli rekordy MX już istnieją i chcesz je zmienić, kliknij przycisk `...`{.action} po prawej stronie każdego wiersza w tabeli odpowiadającego rekordowi, a następnie kliknij `Zmień rekord`{.action}.
+- Jeśli rekord MX nie jest obecny, kliknij przycisk `Dodaj rekord`{.action} po prawej stronie tabeli i wybierz `MX`{.action}. Uzupełnij wymagane informacje w zależności od wybranego rozwiązania poczty elektronicznej:
 
-- **jeśli używasz rozwiązania poczty elektronicznej OVHcloud**: wykorzystaj informacje opisane w [etapie 2: znajomość konfiguracji MX OVHcloud](/pages/web_cloud/domains/dns_zone_mx#etap-2-znajomosc-konfiguracji-mx-ovh){.external};
+**Jeśli dysponujesz rozwiązaniem e-mail OVHcloud**, zapoznaj się z informacjami podanymi w etapie "[Wiedza na temat konfiguracji MX OVHcloud ](#mxovhcloud)".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pl-pl.md]:
```suggestion
**Jeśli dysponujesz rozwiązaniem e-mail OVHcloud**, zapoznaj się z informacjami podanymi w etapie "[Wiedza na temat konfiguracji MX OVHcloud](#mxovhcloud)".
```

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 11]
```diff
@@ -1,83 +1,104 @@
...
 
 ## Requisitos
 
-- Ter acesso à secção de gestão do domínio na [Área de Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt){.external}.
-- Ter acesso à [Área de cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt){.external}.
-- O domínio do e-mail tem de usar a configuração da OVHcloud (os servidores DNS da OVHcloud).
+- Ter acesso à gestão da zona DNS do domínio na [Área de Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt).
+- Ter acesso à [Área de Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt).
+- O domínio em questão deve utilizar a configuração da OVHcloud (ou seja, os servidores DNS da OVHcloud).
+- Dispor de uma oferta MX Plan (incluída na oferta de [alojamento web](https://www.ovhcloud.com/fr/web-hosting/), no [alojamento gratuito 100M](https://www.ovhcloud.com/fr/domains/free-web-hosting/) ou na oferta MX Plan encomendada separadamente), uma das nossas [ofertas de e-mail OVHcloud](https://www.ovhcloud.com/fr/emails/), ou um serviço de e-mail externo.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]: FR links

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 12]
```diff
@@ -1,83 +1,104 @@
...
+>
+> ![email](images/email-dns-conf-mx00.png){.thumbnail}
 
 ## Instruções
 
-### 1 - Informações básicas sobre o campo MX
+### Compreender a função dos registos MX 
 
-Um campo MX associa o seu domínio ao servidor do seu fornecedor de e-mail (a OVHcloud, por exemplo). Quando um dos seus contactos lhe envia um e-mail, o servidor que realiza o envio sabe para que servidor o deve reencaminhar graças ao campo MX.
+Os registos MX (****Mail ****eXchange) permitem associar o seu domínio aos servidores de e-mail de receção associados ao seu serviço de e-mail. Vamos citar um exemplo.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]:
```suggestion
Os registos MX (**M**ail e**X**change) permitem associar o seu domínio aos servidores de e-mail de receção associados ao seu serviço de e-mail. Vamos citar um exemplo.
```

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 13]
```diff
@@ -1,83 +1,104 @@
...
-| ------------------ | ---- | --------------- | ---------- | ----------------- |
-| _deixar em branco_ | 3600 | MX              | 1          | mx0.mail.ovh.net. |
-| _deixar em branco_ | 3600 | MX              | 5          | mx1.mail.ovh.net. |
-| _deixar em branco_ | 3600 | MX              | 50         | mx2.mail.ovh.net. |
-| _deixar em branco_ | 3600 | MX              | 100        | mx3.mail.ovh.net. |
-| _deixar em branco_ | 3600 | MX              | 200        | mx4.mail.ovh.net. |
+> [!warning]
+>
+> De forma geral, **alterar os registos MX na zona DNS do domínio é uma operação delicada** : uma manipulação incorreta pode impossibilitar a receção de e-mails nos seus endereços. Por isso, sugerimos que tenha atenção durante a realização desta operação.
+> Em caso de dúvida, sugerimos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/fr/directory/).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]: FR link

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 14]
```diff
@@ -1,83 +1,104 @@
...
 
-Deve utilizar estes diferentes campos MX na configuração DNS do seu domínio. O passo seguinte permite-lhe efetuar esta operação na configuração DNS da OVHcloud do domínio.
+### Valores da configuração MX da OVHcloud <a name="mxovhcloud"></a>
 
-### 3 - Alterar a configuração de um campo MX da OVHcloud
+Consulte abaixo a configuração MX da OVHcloud que deve utilizar nas soluções MX Plan (só ou incluída numa oferta de [alojamento web da OVHcloud](https://www.ovhcloud.com/fr/web-hosting/)), [E-mail Pro](https://www.ovhcloud.com/fr/emails/email-pro/) e [Exchange](https://www.ovhcloud.com/fr/emails/). Os nossos servidores de e-mail dispõem de um antisspam e antivírus integrado.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]: FR links

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 15]
```diff
@@ -1,83 +1,104 @@
...
 
-- **se possuir uma solução de e-mail da OVHcloud**: consulte as informações no passo [“2 - Conhecer a configuração MX da OVHcloud”](/pages/web_cloud/domains/dns_zone_mx#2-conhecer-a-configuracao-mx-da-ovh){.external};
+![dnsmxrecord](images/mx-records-dns-zone.png){.thumbnail}
 
-- **se possuir outra solução de e-mail**: consulte as informações comunicadas pelo seu fornecedor de serviço de e-mail.
+- Se já existirem e pretender alterá-los, clique no botão `...`{.action} à direita de cada linha da tabela em causa e, a seguir, em `Alterar entrada`{.action}.
+- Se não existir nenhum registo MX presente, clique no botão `Adicionar uma entrada`{.action} à direita da tabela e selecione `MX`{.action}. Introduza as informações necessárias em função da solução de e-mail selecionada:
 
-Depois de preencher os dados, finalize os passos e clique em `Validar`{.action}.
+**Se dispõe de uma solução de e-mail OVHcloud**, consulte as informações fornecidas na etapa « [Conhecer a configuração MX da OVHcloud ](#mxovhcloud) ».
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]:
```suggestion
**Se dispõe de uma solução de e-mail OVHcloud**, consulte as informações fornecidas na etapa "[Conhecer a configuração MX da OVHcloud](#mxovhcloud)".
```

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 16]
```diff
@@ -1,83 +1,104 @@
...
+[Configurar um registo SPF no domínio](/pages/web_cloud/domains/dns_zone_spf)
 
-[Como editar a minha zona DNS?](/pages/web_cloud/domains/dns_zone_edit){.external}
+[Configurar um registo DKIM](/pages/web_cloud/domains/dns_zone_dkim)
 
-Para serviços especializados (referenciamento, desenvolvimento, etc), contacte os [parceiros OVHcloud](https://partner.ovhcloud.com/pt/).
+Para serviços especializados (referenciamento, desenvolvimento, etc.), contacte os [parceiros OVHcloud](https://partner.ovhcloud.com/fr/).
 
-Se pretender usufruir de uma assistência na utilização e na configuração das suas soluções OVHcloud, consulte as nossas diferentes [ofertas de suporte](https://www.ovhcloud.com/pt/support-levels/).
+Se pretender beneficiar de uma assistência ao uso e à configuração das suas soluções OVHcloud, sugerimos que consulte as nossas diferentes [ofertas de suporte](https://www.ovhcloud.com/fr/support-levels/).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]: FR link

File: pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md [hunk 17]
```diff
@@ -1,83 +1,104 @@
...
+[Editar uma zona DNS da OVHcloud](/pages/web_cloud/domains/dns_zone_edit)
 
-[Alterar os servidores DNS de um nome de domínio OVHcloud](/pages/web_cloud/domains/dns_server_general_information){.external}.
+[Configurar um registo SPF no domínio](/pages/web_cloud/domains/dns_zone_spf)
 
-[Como editar a minha zona DNS?](/pages/web_cloud/domains/dns_zone_edit){.external}
+[Configurar um registo DKIM](/pages/web_cloud/domains/dns_zone_dkim)
 
-Para serviços especializados (referenciamento, desenvolvimento, etc), contacte os [parceiros OVHcloud](https://partner.ovhcloud.com/pt/).
+Para serviços especializados (referenciamento, desenvolvimento, etc.), contacte os [parceiros OVHcloud](https://partner.ovhcloud.com/fr/).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_mx/guide.pt-pt.md]: FR link

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**